napi/parser/CHANGELOG.md

```
@@ -4,6 +4,52 @@ All notable changes to this package will be documented in this file.
+
+### Refactor
+
+- 050ecd9 ast/estree: Remove custom serializer for `TSMappedTypeModifierOperator` (#10747) (overlookmotel)
+- a2ba7c3 napi/parser: Add comments about hashbang comments (#10692) (overlookmotel)
+- 3b6d52d napi/parser: Move generated deserializer files (#10481) (overlookmotel)
+
+### Testing
+
+- 14c4bbb ast/estree: Fix raw transfer tests (#10666) (overlookmotel)
+
+## [0.68.0] - 2025-05-03
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)